juniper_official/chassis/check-routing-engine-inlet-temperature.rule

```diff
@@ -0,0 +1,103 @@
+/*
+ * Monitors routing-engine inlet temperature
+ * 
+ * Two inputs control detection
+ *
+ *   1) "re-slot-no" is a regular expression that matches the routing engine
+ *      that you would like to monitor.  By default it's '0-1', which matches
+ *      both the routing engines. Use something like '0' to match only
+ *      routing engine 0.
+ *
+ *   2) "re-inlet-name" is the regular expression that matches the inlet name
+ *      It will match the names 'CPU Inlet1' or 'Inlet temperature'  
+ *      
+ *
+ */
+healthbot {
+  topic hardware.chassis {
+    rule check - routing - engine - inlet - temperature {
+      /*
+       * Monitors the routing-engine inlet temperature  
+       *
+       * Use routing-engine as key for rule.
+       */
+      keys[routing - engine routing - engine - inlet - name];
+      synopsis "Routing engine temperature";
+      description "Collects system RE inlet Temperature statistics periodically";
+      /*
+       * Sensor configuration to get data from network devices.
+       */
+      sensor components - oc {
+        synopsis "routing-engine inlet temperature KPI";
+        description "open-config sensor to collect telemetry data from network device";
+        open - config {
+          sensor - name / components / component;
+          frequency 60 s;
+        }
+      }
+      field routing - engine - inlet - name {
+        sensor components - oc {
+          where "/components/component/properties/property/@name =~ /{{re-inlet-name}}/";
+          path "/components/component/properties/property/@name";
+        }
+        type string;
+        description "Inlet name";
+      }
+      field routing - engine - inlet - temperature {
+        sensor components - oc {
+          where "/components/component/properties/property/@name =~ /{{re-inlet-name}}/";
+          path / components / component / properties / property / state / value;
+        }
+        type integer;
+        description "Inlet temperature";
+      }
+      field routing - engine {
+        sensor components - oc {
+          where "/components/component/@name =~ /^Routing Engine[{{re-slot-no}}]*$/";
```

Comment:
why is a * required here? Please be careful with the * as it means 0 or more occurrences and if I wasn't RE1 I don't have a way to specify on 1 as * takes RE1 RE11 and so on...

Reply:
This is done as similar to "**check-routing-engine-temperature.rule**"

Reply:
Removed * from the check